framework/src/util/SharedLibrary.cpp

```diff
@@ -124,6 +123,21 @@ void SharedLibrary::Load(int flags)
 #endif
 }
 
+void SharedLibrary::Load(const std::unordered_map<std::string, Any>& config)
+{
+  int options = 0;
+#ifdef US_PLATFORM_POSIX
+  try {
+    options = any_cast<int>(config.at(Constants::LIBRARY_LOAD_OPTIONS));
```

Comment:
I think we should do this in `CoreBundleContext::Init()` and save the int options in a member variable. And instead of silently using defaults in case of an exception, we should explicitly check the Any type and log (via `DIAG_LOG`) that the expected type is wrong and we fall back to defaults.

Reply:
Done.